fake_xml_http_request.js

```diff
@@ -160,7 +217,7 @@ FakeXMLHttpRequest.HEADERS_RECEIVED = 2;
 FakeXMLHttpRequest.LOADING = 3;
 FakeXMLHttpRequest.DONE = 4;
 
-FakeXMLHttpRequest.prototype = {
+FakeXMLHttpRequest.extend = {
```

Comment:
I would not attach this object to the FakeXMLHttpRequest object. It seems like it's only needed to loop through and assign on FakeXMLHttpRequest.prototype?